BossMod/Modules/Shadowbringers/HuntA/Sugaar.cs

```diff
@@ -88,7 +88,7 @@ public override void OnCastStarted(BossModule module, Actor caster, ActorCastInf
         public override void OnCastFinished(BossModule module, Actor caster, ActorCastInfo spell)
         {
             if ((AID)spell.Action.ID is AID.NumbingNoiseRotating or AID.TailSnapRotating)
-                AdvanceSequence(caster.Position, spell.Rotation, module.WorldState.CurrentTime);
+                AdvanceSequence(0, module.WorldState.CurrentTime);
```

Comment:
I do not have a replay at this point, i just added it as a safety precaution since sequences breaking at gurangatch and acheloios (even if replay was not provided by the discord user) were observed. Since there can only be one sugaar at a time I guess it does not have any downsides?
I will add the sequences count